nntrainer/src/pooling2d_layer.cpp

```diff
@@ -168,6 +168,19 @@ int Pooling2DLayer::setSize(int *size, PropertyType type) {
   return status;
 }
 
+void Pooling2DLayer::setBatch(unsigned int batch) {
+  input_dim.setTensorDim(0, batch);
```

Comment:
use `Layer::setBatch(batch)` than having the same implementation here.

Reply:
done.